plugins/janus_videoroom.c

```diff
@@ -1916,6 +1948,7 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 		packet.timestamp = ntohl(packet.data->timestamp);
 		packet.seq_number = ntohs(packet.data->seq_number);
 		/* Go */
+		janus_mutex_lock(&participant->subscribers_mutex);
```

Comment:
It fixes [this crash](https://github.com/meetecho/janus-gateway/pull/403#issuecomment-248583617) reported by @cacheworks.  [Locks in glibc were improved](https://lwn.net/Articles/534758/) a few years ago so that this may not have too much of a performance impact.